Traces support (#594)
grafana/azure-data-explorer-datasource: README.md

  | extend series_decompose_forecast(AvgHatInventory, 30) | project-away *residual, *baseline, *seasonal
```

### Trace Visualization Support

> The trace format option can be used to display appropriately formatted data using the built in trace visualization. In order to use this visualization, data must be presented following the schema that is defined [here](https://grafana.com/docs/grafana/latest/explore/trace-integration/#data-frame-structure). The schema contains the `logs`, `serviceTags`, and `tags` fields which are expected to be JSON objects. These fields will be converted to the expected data structure provided the schema in ADX matches the below:

Reviewer: ```suggestion
The trace format option can be used to display appropriately formatted data using the built in trace visualization. To use this visualization, data must be presented following the schema defined [here](https://grafana.com/docs/grafana/latest/explore/trace-integration/#data-frame-structure). The schema contains the `logs`, `serviceTags`, and `tags` fields which are expected to be JSON objects. These fields will be converted to the expected data structure provided the schema in ADX matches one of these formats:
```

## Query Builder - Data Types

> The query builder provides an easy to use interface to query Azure Data Explorer. As of v4.1.0 columns of type `dynamic` are also appropriately supported within the query builder. Dynamically typed columns can now be queried using the `Where`, `Aggregate`, and `Group By` operations. When choosing one of these operations the options will be appropriately populated based on the values within the dynamic column. This encompasses arrays, JSON objects, and nested objects within arrays. There are, however, some limitations; as only the first 50,000 rows are queried for data, only properties contained within the first 50,000 rows will be listed as options in the builder selectors. Also, due to the fact that these queries make use of `mv-expand` they may become resource intensive.

Reviewer: ```suggestion
The query builder provides an easy to use interface to query Azure Data Explorer. As of v4.1.0, columns of type `dynamic` are also appropriately supported within the query builder. Dynamically typed columns can now be queried using the `Where`, `Aggregate`, and `Group By` operations. When choosing one of these operations, the options will be populated based on the values within the dynamic column. This encompasses arrays, JSON objects, and nested objects within arrays. A limitation is only the first 50,000 rows are queried for data, so only properties contained within the first 50,000 rows will be listed as options in the builder selectors. Also, due to the fact that these queries make use of `mv-expand`, they may become resource intensive.
```
